canonical-web-and-design/ubuntu.com
# /azure

## Done

Added /azure section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/azure
- Ensure that the modal contact form works
-  http://0.0.0.0:8001/azure/pro
-  http://0.0.0.0:8001/azure/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)


## Issue / Card

Fixes #2177
Fixes #2178
